app/controllers/api/v2/chapters_controller.rb

```diff
@@ -55,8 +73,16 @@ def find_chapter
         raise Sequel::RecordNotFound if @chapter.goods_nomenclature_item_id.in? HiddenGoodsNomenclature.codes
       end
 
+      attr_reader :chapter
+
       def chapter_id
-        "#{params[:id]}00000000"
+        c_id = params[:id]
+
+        if c_id.length == 2
+          "#{c_id}00000000"
+        else # it is single digit
```

Comment:
I think we should just stick with the short code format to fetch the chapter headings

Reply:
I've simplified this, I think it's handy to have.

If you try this URL on production:
https://staging.trade-tariff.service.gov.uk/chapters/1
you would get "not found", which is weird since chapter 01 does actually exist.

Reply:
We definitely don't want to change the protocol for representations of the goods nomenclature item id to start ignoring the first 0. This is a commonly understood way of representing the chapter in the harmonised system. If we change it we're adding complexity/create a non-conformant api around what are essentially worldwide standards.

Reply:
Even in the short form its actually the wrong thing to do